examples/training/classification/imagenet/basic_training.py

```diff
@@ -193,6 +192,11 @@ def augment(img, label):
     inputs = {"images": img, "labels": label}
     for layer in AUGMENT_LAYERS:
         inputs = layer(inputs)
+    if tf.random().uniform(()) > 0.5:
```

Comment:
This should just be `tf.random.uniform()`

Reply:
Whoops, good catch!